zkevm-circuits/src/util/cell_manager_strategy.rs

```diff
@@ -17,7 +17,7 @@ impl CMFixedWidthStrategyDistribution {
         }
     }
 
-    pub(crate) fn get(&self, cell_type: CellType) -> Option<&Vec<Column<Advice>>> {
+    pub(crate) fn _get(&self, cell_type: CellType) -> Option<&Vec<Column<Advice>>> {
```

Comment:
If we prefix with underscore, reader might still wondering the reason.

So I suggest we turn to have explicitly `allow(dead_code, reason = "xxx")` in this method.

```rust
#[allow(dead_code, reason = "this method will be used in XXX")]
pub(crate) fn get(&self, cell_type: CellType) -> Option<&Vec<Column<Advice>>> {
```